ha/setup/ha_setup.py

```diff
@@ -401,6 +401,12 @@ def process(self):
             self._execute.run_cmd(f"setfacl -R -m g:{const.USER_GROUP_ROOT}:rwx {const.RA_LOG_DIR}")
             self._execute.run_cmd(f"setfacl -R -m g:{const.USER_GROUP_HACLIENT}:r-x {const.CONFIG_DIR}")
             self._execute.run_cmd(f"setfacl -R -m g:{const.USER_GROUP_ROOT}:rwx {const.CONFIG_DIR}")
+            for file_path in const.RA_LOG_FILES:
+                if file_path and not os.path.exists(file_path):
+                    os.makedirs(os.path.dirname(file_path), exist_ok=True)
+                    open(file_path, 'x')
+                self._execute.run_cmd(f"setfacl -R -m g:{const.USER_GROUP_HACLIENT}:rwx {file_path}")
```

Comment:
This can be done in ConfigManager.init
That way new log files need not be created here.
Also scaling in case of new logs will not be an issue.

Reply:
I tried adding it in the ConfigManager.init, But when we try to instantiate SimpleCommand class it fails at Log.debug statement with none value exception.